connect/json/src/test/java/org/apache/kafka/connect/json/JsonConverterTest.java

```diff
@@ -607,6 +611,19 @@ public void testCacheSchemaToJsonConversion() {
         assertEquals(2, cache.size());
     }
 
+    @Test
+    public void testJsonSchemaCacheSizeFromConfigFile() throws URISyntaxException, IOException {
+        URL url = getClass().getResource("/connect-test.properties");
```

Comment:
This works fine, though if you wanted to simplify, the key thing here is just that you are using a `Map<String, String>`, so you could get rid of the resource file and just create it on the fly:

```
Collections.singletonMap("schemas.cache.size", "1");
```